### ISSUE TITLE
slide images on index page are broken

### ISSUE THREAD

**Issue by gjost** [densho/ddr-public#36]
All the slide images on http://ddrstage.densho.org/ are broken.

**Comment by gjost** [comment]
Looks like somebody hard-coded the domain name in there:
```
{% homeslideitem "/ddr/densho/2/33" "http://ddr.densho.org/media/ddr-densho-2/ddr-densho-2-33-mezzanine-16fe864756-a.jpg" %}
{% homeslideitem "/ddr/densho/266/9" "http://ddr.densho.org/media/ddr-densho-266/ddr-densho-266-9-mezzanine-818eb5aa3e-a.jpg" %}
```

**Comment by gjost** [comment]
Fixed in cabe8e7.
